pyrealm/splash/const.py

```diff
@@ -0,0 +1,32 @@
+"""TODO - merge into existing constants classes."""  # noqa: D210, D415
+
+
+import numpy
+
```

Comment:
Are you planning to add typing?

Reply:
Planning to _completely_ replace this with constants dataclasses with typing.  The issue is that I need to identify universal constants and split up PModel constants. Sound familiar?